go-controller/pkg/libovsdb/util/router.go

```diff
@@ -60,3 +60,11 @@ func CreateDefaultRouteToExternal(nbClient libovsdbclient.Client, nodeName strin
 	}
 	return nil
 }
+
+func GetLogicalRouterSNATConntrackZone(logicalRouterName string) (string, error) {
+	stdout, stderr, err := util.RunOVSVsctl("get", "bridge", "br-int", fmt.Sprintf("external_ids:ct-zone-%s_snat", logicalRouterName))
```

Comment:
oopsy we don't do ctl calls like :)
can you please express this as libovsdb abstraction...?

Reply:
Wait, what do you mean with libovsdb abstraction ? we don't have Model for ovs "Bridge" and friends.

Reply:
Moving this under `utils/ovs.go`, there are similar calls there

Reply:
oh "get" "bridge"... hmm I'd think there should be some other way to get this GR's zone info from NBDB via datapath command? @dceara : any ideas here?

Reply:
The zone ID is stored in the local OVS database, in the bridge table external ids.  That vsctl call seems correct to me and on the node side we still use ovs-vsctl AFAICT (https://github.com/ovn-org/ovn-kubernetes/pull/3616 was going to address that).

Reply:
Let's keep it then under `utils/ovs.go` for now, it will be moved with the rest as part of https://github.com/ovn-org/ovn-kubernetes/pull/3616 in the future I suppose.